### PR TITLE
I finished the assignment.

### PR DESCRIPTION
**Pull request by yuki-sato-0402**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -64,10 +62,34 @@ void variableDeclarations()
...
+    
+
+    unsigned int num10 = 4;
+    unsigned int num11 = 5;
+    unsigned int num12 = 6;
+ 
 
     
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(number,num,num2,num3,num4,num5,num6,A,B,C,num7,num8,num9,chr1,chr2,chr3,num10,num11,num12); //passing each variable declared to the ignoreUnused() function
```
Comment (matkatmusic) [inline main.cpp:92]: It is important to be consistent when formatting your code.
The coding style for this course asks that you add a space after all commas.
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -83,43 +105,101 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 /*
  1)
- */
+ */
+
+int num(int num1 = 1, int num2 = 2 )
+{
+    ignoreUnused(num1, num2 );
```
Comment (matkatmusic) [inline main.cpp:112]: It is important to be consistent when formatting your code.
parentheses should not have spaces on either side of them.
```
int num(int num1 = 1, int num2 = 2)
{
    ignoreUnused(num1, num2); 
```
- [ ] revise

File: main.cpp [hunk 3]
```diff
@@ -83,43 +105,101 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
 
 /*
  2)
- */
+ */
+
+int cost (int grade = 1, int size = 2, bool specialOption =true)
+{
+    ignoreUnused(grade ,size, specialOption); 
+    return{};
```
Comment (matkatmusic) [inline main.cpp:123]: Code style:
- put spaces around `=`
- commas should have a space after them, not before them.
- parentheses should not have spaces on either side of them.

- [ ] revise

File: main.cpp [hunk 4]
```diff
@@ -83,43 +105,101 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+}
 
 /*
  3)
- */
+ */
+int numberOfAnimals(int dog, int cat  )
+{
+    ignoreUnused(dog, cat );
```
Comment (matkatmusic) [inline main.cpp:131]: Code style:
- parentheses should not have spaces on either side of them.

- [ ] revise

File: main.cpp [hunk 5]
```diff
@@ -83,43 +105,101 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+}
 
 /*
  4)
- */
+ */
+
+bool score(float score1=45.6f , float score2 =67.9f  )
+{
+    ignoreUnused(score1, score2 );
```
Comment (matkatmusic) [inline main.cpp:141]: Code style:
- put spaces around `=`
- commas should have a space after them, not before them.
- parentheses should not have spaces on either side of them.

- [ ] revise

File: main.cpp [hunk 6]
```diff
@@ -83,43 +105,101 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+}
 
 /*
  5)
- */
+ */
+
+bool select(int money=300  , float gold =25.5  )
+{
+    ignoreUnused(money, gold );
```
Comment (matkatmusic) [inline main.cpp:151]: Code style:
- put spaces around `=`
- commas should have a space after them, not before them.
- parentheses should not have spaces on either side of them.

- [ ] revise

File: main.cpp [hunk 7]
```diff
@@ -83,43 +105,101 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+}
 
 /*
  6)
- */
+ */
+
+float calculation(float num3= 55.6f ,float num4= 43.6f)
+{
+    ignoreUnused(num3, num4 );
```
Comment (matkatmusic) [inline main.cpp:161]: Code style:
- put spaces around `=`
- commas should have a space after them, not before them.
- parentheses should not have spaces on either side of them.

- [ ] revise

File: main.cpp [hunk 8]
```diff
@@ -83,43 +105,101 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+}
 
 /*
  7)
  */
-
+
+void printNum ( int num5 =54  ,float num6 = 44.2f)
+{
+    ignoreUnused(num5, num6 );
```
Comment (matkatmusic) [inline main.cpp:171]: Code style:
- put spaces around `=`
- commas should have a space after them, not before them.
- parentheses should not have spaces on either side of them.

- [ ] revise

File: main.cpp [hunk 9]
```diff
@@ -83,43 +105,101 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+{
+    ignoreUnused(num5, num6 ); 
+}
 /*
  8)
  */
-
+void printChar (char chr1= 'a', char chr2= 'b')
+{
+    ignoreUnused(chr1, chr2 );
```
Comment (matkatmusic) [inline main.cpp:178]: Code style:
- put spaces around `=`
- commas should have a space after them, not before them.
- parentheses should not have spaces on either side of them.

- [ ] revise

File: main.cpp [hunk 10]
```diff
@@ -83,43 +105,101 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    ignoreUnused(chr1, chr2 ); 
+}
 /*
  9)
- */
+ */
+
+float priceOfLand (float pricePerUnitOfArea= 33.5f,float landSize= 42.1f)
+{
+    ignoreUnused(pricePerUnitOfArea, landSize );
```
Comment (matkatmusic) [inline main.cpp:186]: Code style:
- put spaces around `=`
- commas should have a space after them, not before them.
- parentheses should not have spaces on either side of them.

- [ ] revise

File: main.cpp [hunk 11]
```diff
@@ -83,43 +105,101 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+
 
 /*
  10)
- */
+ */
+
+float mileage (float distance =78.3f, float fuelConsumption = 4.3f)
+{
+    ignoreUnused(distance,fuelConsumption );
```
Comment (matkatmusic) [inline main.cpp:197]: Code style:
- put spaces around `=`
- commas should have a space after them, not before them.
- parentheses should not have spaces on either side of them.

- [ ] revise

File: main.cpp [hunk 12]
```diff
@@ -141,27 +221,27 @@ int main()
...
+    printChar ('a' , 'b');
     //9)
-    
+    auto  autoLand =priceOfLand (33.5f ,42.1f );
     //10)
+    auto car =mileage (78.3f,4.3f);
     
-    
-    ignoreUnused(carRented);
+    ignoreUnused(carRented,count,purchase,animals,compare,whichIsExpensive,calaculated,printNum, printChar,autoLand,car);
```
Comment (matkatmusic) [inline main.cpp:244]: Code style:
- put spaces around `=`
- commas should have a space after them, not before them.
- parentheses should not have spaces on either side of them.

- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
Your submission is OK, excluding the code formatting issues marked below

File: main.cpp (hunk 12 above)
Comment (yuki-sato-0402) [reply main.cpp]: I have not been able to correct this issue.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**